index.html

```diff
@@ -50,7 +53,179 @@ <h2>Software Developer</h2>
+    <hr class="horizontal-line" />
+
+    <!--about sections-->
+    <section class="about-section">
+      <div>
+        <h3 class="title">Languages</h3>
+        <ul class="skills">
+          <li>Javascript</li>
+          <li>Python</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </div>
+
+      <div>
+        <h3 class="title">Frameworks</h3>
+        <ul class="skills">
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+
+      <div>
+        <h3 class="title">Skills</h3>
+        <ul class="skills">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <!--about sections ends here-->
+
+    <hr class="horizontal-line" />
+
+    <!--section for social links and icons-->
+    <!--webhint flags section tags without heading as error.
+    Hence div tag is used to wrap the icons to pass the webhint check.
+    This approach is used since there is no heading given in the figma design on figma-->
+    <div class="social-icons">
+      <a href="#"><i class="fab fa-github"></i></a>
+      <a href="#"><i class="fab fa-linkedin-in"></i></a>
+      <a href="#"><i class="fab fa-instagram"></i></a>
+      <a href="#"><i class="fab fa-twitter"></i></a>
+      <a href="#"><i class="fab fa-facebook"></i></a>
+    </div>
+    <!--section for social links and icons ends here-->
+
+    <!--work section-->
+    <section class="work-section">
+      <div class="work-section-title-container">
+        <i class="fa fa-arrow-down work-icon"></i>
+        <h3 class="work-section-title">Work</h3>
+      </div>
+
+      <h1 class="recent-work">MY RECENT WORK</h1>
+      <div class="work-section-hr">
+        <hr class="horizontal-line" />
+      </div>
+
+      <!--project template 1-->
+      <div class="project">
+        <div class="project-image-container">
+          <img
+            src="./assets/images/background.jpg"
+            alt="project image"
+            class="project-image"
+          />
+        </div>
+
+        <div>
+          <h2 class="project-name">Multi-Post Stories</h2>
+        </div>
+
+        <div>
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+
+        <div class="technologies">
+          <p>css</p>
+          <p>html</p>
+          <p>bootstrap</p>
+          <p>ruby</p>
+        </div>
+
+        <div>
+          <button type="button" class="project-button">See Project</button>
+        </div>
+      </div>
+      <!--project template 1 ends here-->
```

Comment:
- [ ] To stick to the design you should add a background image on projects. Check the original design.
- [ ] Also please change font-family and color too. It is different from the original.
**_I select the first project section. Please apply it to other projects in the page._**
### ORIGINAL DESIGN
![image](https://user-images.githubusercontent.com/38635158/148597406-bfd4b411-143d-416a-9f91-b28df2db5868.png)
___
### YOUR DESIGN
![image](https://user-images.githubusercontent.com/38635158/148597608-18382e21-c6fe-4dd1-a50c-84da9fca89dd.png)

Reply:
- background images added successfully
- Inter font added as well